### ISSUE TITLE
None docker images built since a month

### ISSUE THREAD

**Issue by llaville** [devilbox/docker-php-fpm#195]
Hello @cytopia 

Don't know, if you're noticed, but none docker images are not built since there is a syntax error in pip installation script.

See https://github.com/devilbox/docker-php-fpm/runs/1859491000?check_suite_focus=true#step:10:3469 

I've rebuilt locally (base, mods, and prod images without error, but it's not possible to get work up-and-ready).

Could you have a look on this issue please ?
Thanks 

PS: latest Github Actions running is  https://github.com/devilbox/docker-php-fpm/actions/runs/464866293

See origin of syntax error:  https://github.com/pypa/pip/blame/master/src/pip/_internal/cli/main.py#L60

**Comment by llaville** [comment]
While working on PR #196, I've found a typo error in README file : `generate` target in Makefile does not exists

```
diff --git a/build/README.md b/build/README.md
index c5f7409..d2a4537 100644
--- a/build/README.md
+++ b/build/README.md
@@ -17,7 +17,7 @@ ansible-playbook generate.yml --diff
 ```bash
 # From inside root git directory
 cd ..
-make generate
+make gen-dockerfiles
 ```

 **Requirements**
```

**Comment by llaville** [comment]
I'll close this issue, because solution with fixes are available in #199 report
